tests/unittests/clustering/test_dunn_index.py

```diff
@@ -0,0 +1,84 @@
+# Copyright The Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from functools import partial
+from itertools import combinations
+
+import numpy as np
+import pytest
+from torchmetrics.clustering.dunn_index import DunnIndex
+from torchmetrics.functional.clustering.dunn_index import dunn_index
+
+from unittests.clustering.inputs import (
+    _single_target_intrinsic1,
+    _single_target_intrinsic2,
+)
+from unittests.helpers import seed_all
+from unittests.helpers.testers import MetricTester
+
+seed_all(42)
+
+
+def _np_dunn_index(x, labels, p):
```

Comment:
```suggestion
def _np_dunn_index(data, labels, ord):
```
lets be more descriptive

Reply:
Had to switch `ord` back to `p` since `ord` is a python builtin.